Game.h

```diff
@@ -98,9 +98,18 @@ class Game
     void prepareScene(RGB={255/3,255/3,255/3});
     void presentScene();
     // control logic (game.cpp)
+    constexpr static uint32_t DELAY = 16;
     void initSystems();
     void processInput();
     void gameLoop();
+    // timer logic
+    int level = 1;
+    int timeLeft = 0;
+    bool fastFall = false;
+    // dropDelay is in terms of DELAY increments.
+    int dropDelay() const {
+        return pow(0.8-(level-1)*0.007,level-1)*1000/DELAY;
+    };
```

Comment:
`dropDelay` is actually incorrect, I believe. There should be some conversion from 'G' (which is what the `dropDelay` formula uses) to ms, frames, or seconds, which would allow for a more faithful simulation.

Reply:
I think G refers to "per frame", which appears to match with the current delay. Which means I think I implemented it to be 16 times faster than it should have been?